diff -Naur cdparanoia-III-alpha9.7/configure.in cdparanoia-III-alpha9.7-p/configure.in
--- cdparanoia-III-alpha9.7/configure.in	Mon Aug 16 00:29:43 1999
+++ cdparanoia-III-alpha9.7-p/configure.in	Mon Jul 17 16:32:30 2000
@@ -14,21 +14,19 @@
 
 if test -z "$GCC"; then
 	DEBUG="-g"
-	OPT="-O"
+	OPT=""
 else
 	case $host in 
 	i?86-*-linux*)
 		DEBUG="-g -Wall -fsigned-char"
-		OPT="-O20 -ffast-math -fsigned-char -finline-functions";;
+		OPT="-ffast-math -fsigned-char -finline-functions";;
 	*)
 		DEBUG="-g -Wall -fsigned-char"
-		OPT="-O20 -fsigned-char";;
+		OPT="-fsigned-char";;
         esac
 fi
 
 AC_HEADER_STDC
-
-CFLAGS=""
 
 TYPESIZES="" 
 
