diff -Nur grip-2.1.orig/Makefile grip-2.1/Makefile
--- grip-2.1.orig/Makefile	Mon May 31 09:47:25 1999
+++ grip-2.1/Makefile	Thu Jun  3 03:56:08 1999
@@ -21,9 +21,7 @@
 LIBS=	`gtk-config --libs` -lpthread
 
 # This is needed for "make install"
-OWNER  = root
-GROUP  = root
-INSTALL  = /usr/bin/install -o $(OWNER) -g $(GROUP)
+INSTALL  = /usr/bin/install
 
 
 # ----------- You shouldn't need to make changes below here. -------------
